scripts/preinstall.sh

```diff
@@ -1,8 +1,22 @@
 #!/usr/bin/env bash
 
+function cocoapods() {
+    echo "Installing Cocoapods"
+    if [[ $(uname -p) == 'arm' ]]; then
+      npm run ios-gems-m1 &> /dev/null || exit 1
+    else
+      npm run ios-gems &> /dev/null  || exit 1
+    fi
+    
+}
```

Comment:
@enahum Heads up that this addition seems to have broken something on Linux.

```
> mattermost-mobile@2.3.0 preinstall
> sh scripts/preinstall.sh && npx solidarity

scripts/preinstall.sh: 3: Syntax error: "(" unexpected
```

Reply:
strange, can you check if removing the parentesis on the function works?

Reply:
that is even stranger as it seems to be under an if statement checking for `darwin` which has always been there, unless the function is executing anyway for some reason.

Reply:
that'll be great thanks @streamer45